test/example/c-api-kitchen-sink.c

```diff
@@ -1033,28 +1033,24 @@ void test_for_each() {
       assert(BinaryenGetExportByIndex(module, i) == exps[i]);
     }
 
-    BinaryenAddGlobal(module, "a-global", BinaryenTypeInt32(), 0, makeInt32(module, 125));
+    const int32_t expected_offsets[] = { 10, 125 };
```

Comment:
```suggestion
    const uint32_t expected_offsets[] = { 10, 125 };
```

Reply:
It might be more readable to group all the static data about the segments here at the top instead of mixing it in with binaryen API calls.